php/hyperf/.gitignore

```diff
@@ -0,0 +1,3 @@
+vendor
```

Comment:
could you please add this to `php/.gitignore` ? I prefer to have one `.gitignore` per **language**